gtfs-realtime/proto/gtfs-realtime.proto

```diff
@@ -54,7 +54,8 @@ message FeedMessage {
 // Metadata about a feed, included in feed messages.
 message FeedHeader {
   // Version of the feed specification.
-  // The current version is 2.0.  Valid versions are "2.0", "1.0".
+  // The current version is 2.0. Valid versions are "3.0", "2.0", "1.0".
```

Comment:
I'm not sure we really need a version bump to "v3" as part of this proposal. The proposed changes aren't breaking, and it's really an incremental improvement to TripUpdate. In the Service Changes v3.1 docs the term "TripUpdates v3" was used to differentiate it from some of the other groups of feature, but I don't think it's as relevant for the whole spec. I'd prefer to reserve v3 for breaking changes like requiring more fields in VehiclePositions.

Unless I'm missing something - let me know.

Reply:
The main thing is that feels a bit breaking is that in an existing feed without these fields, `SKIPPED` could be appropriate for the drop-off only scenario as the behavior for a rider-facing application we might expect is to not show predictions for `NO_PICKUP`. With these feeds available, we're imagining that we would keep the relationship `SCHEDULED` since the vehicle _may_ stop, but only upon rider request. 

Agree, though, that there are other cases that feel more additive. For example, a rider application could choose to implement a net new feature to notify a rider on the vehicle upon seeing `MUST_ASK_DRIVER_DROP_OFF ` in the real-time feed.

I did see that there was a bunch of discussion in #221 about the migration path to `DUPLICATED` within the same feed (thank you for documenting that!). I'm not sure where consumers are today, but I'm primarily concerned about time-to-implementation, so I feel that it's safest to maintain two separate feeds, for the time being, and for consumers to adopt the new version once they are ready.

Reply:
Resolving this thread now that the version bump has been removed in favor of a in-place migration guide